distribution/docs/src/main/resources/_contents/managing-alliance-contents.adoc

```diff
@@ -0,0 +1,26 @@
+
+The ${branding-expanded} Application provides support for ingesting and searching for MGMP products.
+
+=== ${branding-expanded} Application Prerequisites
+
+Before the ${branding-expanded} Application can be installed:
+
+* the ${ddf-platform} Application
+* the ${ddf-catalog} Application
+* the ${ddf-spatial} Application must be installed
```

Comment:
This bulleted list reads oddly to me. Maybe if the last line said, "...must all be installed" it would be less awkward.